### PR TITLE
Add QE test mappings for ISC components

### PR DESCRIPTION
**Pull request by xiaojiey**
Update QE test mappings for ISC components Compaliance Operator, File Integrity Operator, Security Profiles Operator, and oc-compliance.
One junit example:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/ginkgo-test/266121/artifact/private/junit_e2e_2024-09-01-23-48-48.xml

### CODE REVIEW

**Comment by xiaojiey** [comment]
@stbenjam Could you please help to review? Thanks.

**Comment by xiaojiey** [comment]
@jhou1  @stbenjam  @xueqzhan Could you please help to review? Thanks.

File: pkg/components/complianceoperator/component.go [hunk 1]
```diff
@@ -12,9 +12,13 @@ type Component struct {
 var ComplianceOperatorComponent = Component{
 	Component: &config.Component{
 		Name:                 "Compliance Operator",
-		Operators:            []string{},
+		Operators:            []string{"compliance operator"},
```
Comment (stbenjam) [inline pkg/components/complianceoperator/component.go:15]: Operators should be a list of cluster operator names

**Comment by stbenjam** [comment]
It doesn't look like this changed any mappings.  We're not importing anything from Jenkins AFAIK.

**Pull request merged**